pkg/logs/sender/sender.go

```diff
@@ -6,21 +6,26 @@
 package sender
 
 import (
+	"context"
+	"github.com/DataDog/datadog-agent/pkg/logs/config"
```

Comment:
nits: this import should be grouped with other `github.com/DataDog/datadog-agent/pkg/logs/` imports